atomkraft/Reactor.py

```diff
@@ -0,0 +1,10 @@
+from os import PathLike
+from typing import List
+
+
+def generate_reactor(
+    actions_list: List[str], variables_list: List[str], stub_file_path: PathLike = None
```

Comment:
`stub_file_path: PathLike = None` should be `stub_file_path: Optional[PathLike] = None` or `stub_file_path: PathLike | None = None`.